omero/developers/index.txt

```diff
@@ -115,6 +115,7 @@ Web
     Web/WritingViews
     Web/WritingTemplates
     Web/CSRF
+    Web/Webclient
```

Comment:
OK. I guess the reason I put it last is that I expect the number of developers who actually want to work on the webclient to be kinda small (why we've not had this well documented before).
Let's see what the page looks like when it's "done". Any opinion @aleksandra-tarkowska?